extensions/sftp/processors/ListSFTP.cpp

```diff
@@ -59,73 +59,85 @@ namespace nifi {
 namespace minifi {
 namespace processors {
 
-core::Property ListSFTP::ListingStrategy(
-    core::PropertyBuilder::createProperty("Listing Strategy")->withDescription("Specify how to determine new/updated entities. See each strategy descriptions for detail.")
-        ->isRequired(true)
-        ->withAllowableValues<std::string>({LISTING_STRATEGY_TRACKING_TIMESTAMPS,
-                                            LISTING_STRATEGY_TRACKING_ENTITIES})
-        ->withDefaultValue(LISTING_STRATEGY_TRACKING_TIMESTAMPS)->build());
-core::Property ListSFTP::RemotePath(
-    core::PropertyBuilder::createProperty("Remote Path")->withDescription("The fully qualified filename on the remote system")
-        ->isRequired(false)->supportsExpressionLanguage(true)->build());
-core::Property ListSFTP::SearchRecursively(
-    core::PropertyBuilder::createProperty("Search Recursively")->withDescription("If true, will pull files from arbitrarily nested subdirectories; "
+core::Property ListSFTP::ListingStrategy(core::PropertyBuilder::createProperty("Listing Strategy")
+    ->withDescription("Specify how to determine new/updated entities. See each strategy descriptions for detail.")
+    ->isRequired(true)
+    ->withAllowableValues<std::string>({LISTING_STRATEGY_TRACKING_TIMESTAMPS, LISTING_STRATEGY_TRACKING_ENTITIES})
+    ->withDefaultValue(LISTING_STRATEGY_TRACKING_TIMESTAMPS)->build());
+
+core::Property ListSFTP::RemotePath(core::PropertyBuilder::createProperty("Remote Path")
+    ->withDescription("The fully qualified filename on the remote system")
+    ->isRequired(false)->supportsExpressionLanguage(true)->build());
+
+core::Property ListSFTP::SearchRecursively(core::PropertyBuilder::createProperty("Search Recursively")
+    ->withDescription("If true, will pull files from arbitrarily nested subdirectories; "
                                                                                  "otherwise, will not traverse subdirectories")
```

Comment:
this should be indented less:
```suggestion
    ->withDescription("If true, will pull files from arbitrarily nested subdirectories; "
                      "otherwise, will not traverse subdirectories")
```